dashboard/package.json

```diff
@@ -16,10 +16,16 @@
     "babel-jest": "^27.5.1",
     "classnames": "^2.3.1",
     "customize-cra": "^1.0.0",
+<<<<<<< HEAD
+=======
+    "crypto-js": "^4.1.1",
+    "dotenv": "^16.0.1",
+>>>>>>> 67cefb6df (Fetch endpoints)
```

Comment:
You have an unresolved merge conflict here!

Reply:
Updated!